src/DynamoCore/Configuration/Context.cs

```diff
@@ -2,14 +2,16 @@
 
 namespace Dynamo.Configuration
 {
-    public static class Context
+    public enum DynamoAppContext
     {
-        // TODO: MAGN-7242
-        public const string NONE = "None";
-        public const string REVIT_2014 = "Revit 2014";
-        public const string REVIT_2015 = "Revit 2015";
-        public const string VASARI_2014 = "Vasari 2014";
+        DynamoStandalone,
```

Comment:
Using product specific context is not a good idea. I think we should have some mechanism to override customizer behavior and that should be implemented by respective product.

Tomorrow if Dynamo For XYZ wants to customize customizer, we can't come back here and add code to detect that product.